### PR TITLE
004 view band roles

### PR DESCRIPTION
**Pull request by JennicaMistry**
Added the ID tags to the Band html page and make sure the band UI tests function - User Story 004

### CODE REVIEW

**Review by hellofadude — CHANGES_REQUESTED** [review]
You should not be committing files in node_modules folder. ATM it says you're trying to commit 5000 plus changes. Please remove changes to node_modules folder and resubmit.

File: app/interface.js [hunk 1]
```diff
@@ -44,9 +44,21 @@ exports.getJobRoles = async () => {
     }
     catch(e) {
         console.log(e);
+
     }
     return("Could not return roles")
 }
+exports.viewBandLevel = async () => {
+    try{
+        const response = await axios.get('http://localhost:8080/api/viewBandLevel');
```
Comment (shaunganley) [inline app/interface.js:53]: shouldn't hardcode localhost, store in an environment variable instead

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: app/config.js [hunk 2]
```diff
@@ -26,6 +27,9 @@ module.exports = {
   useHttps: 'true',
 
   // Enable or disable Browser Sync (local development only)
-  useBrowserSync: 'true'
+  useBrowserSync: 'true',
+
+  bandLevel: process.env.BAND_LEVEL
```
Comment (shaunganley) [inline app/config.js:32]: don't pass in the entire URL, pass in the base URl then append the specific route to each endpoint in the code

File: package.json [hunk 3]
```diff
@@ -1,54 +0,0 @@
-{
```
Comment (shaunganley) [inline package.json:1]: these files shouldn't have been deleted?

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Review by hellofadude — APPROVED** [review]

**Pull request merged**